docs/guide/custom-ui.md

```diff
@@ -1,39 +1,292 @@
+    </span>
+    <span v-else>
+      None
+    </span>
+  </p>
+</template>
+
+<script>
+import AposCellMixin from 'Modules/@apostrophecms/ui/mixins/AposCellMixin';
+
+export default {
+  name: 'AposCellBasic',
+  mixins: [ AposCellMixin ],
+  computed: {
+    hasValue() {
+      return this.get(this.header.name) != null;
+    }
+  },
+  methods: {
+    isActive(index) {
+      return index <= (this.get(this.header.name) || 0);
+    }
+  }
+};
+</script>
+
+```
+  <template v-slot:caption>
+    modules/star-range-field/ui/apos/components/ColumnStarRating.vue
+  </template>
+</AposCodeBlock>
+
+While the Vue component could be placed in any module, it makes the most sense to group it in the `star-range-field` module because it might be used as a column in any piece type, not just a `review`.
+
+This component uses `AposCellMixin` to do two important things:
+
+* The component gains access to the configuration of the column by accessing the `header` prop, which is automatically declared by the mixin.
+* The component can fetch properties from the piece by invoking `this.get` with any property name. Following Apostrophe's convention this method automatically fetches from the published version of the piece if it exists, otherwise from the draft.
+
+## Overriding standard Vue.js components by name
+
+Most of the time we don't need to override admin UI components that ship with Apostrophe. But if we have a need, we can do so by **placing a file with the same name as a standard component in the `ui/apos/components` subdirectory of a project-level module.** You can also do this in a custom npm module to achieve reuse across projects.
+
+Apostrophe will use only the last version of a component that it finds during startup. The general startup order is:
+
+1. Core Apostrophe modules
+2. Installed and project-level modules, in the order they are configured in `app.js`
+that is configured. For instance, if the last module in our project's `app.js` modules list contains a `ui/apos/components/AposLogPadless.vue` file, that logo will be used in the admin bar, in place of the version that is normally loaded from Apostrophe core or in any module configured earlier.
+
+::: note
+For more information about the patterns used, props provided and APIs needed to implement a component, it's necessary to study the source code of the original.
+:::
+
+## Overriding standard Vue.js components through configuration
+
+There can be only one `AposDocsManager` Vue.js component definition in a project, but sometimes we need different behavior for a specific piece type only. We could work around this by overriding a core component and adding conditional logic, but this results in code that is hard to maintain, and also means we are stuck maintaining a copy of a complex component and missing out on bug fixes and improvements. It would be better if we could **specify a different, custom component name to be used** to manage a particular piece type.
```

Comment:
There are a few previous comments that you don't seem to have addressed here.

Remove "Vue.js" from "only one `AposDocsManager` Vue.js component in a project". Not critical, but repetitive.

Remove "only" at the end of the second sentence as it's redundant.

We could work around this by overriding" to "We could override" (then, "and add"). Period after "hard to maintain," then new sentence.

> It would be better if we could...

"Could" isn't necessary since the purpose here is to show they can. Simply "It would be better to specify...".